src/Handler/CurlFactory.php

```diff
@@ -375,6 +375,14 @@ private function applyHandlerOptions(EasyHandle $easy, array &$conf)
             $conf[CURLOPT_TIMEOUT_MS] = $options['timeout'] * 1000;
         }
 
+        if (isset($options['force_ipv4'])) {
+            if (defined('CURLOPT_IPRESOLVE') && defined('CURL_IPRESOLVE_V4')) {
```

Comment:
Id should not be dependent on any PHP version. It just won't work when the curl library is not present.